app/common/gutil.ts

```diff
@@ -948,6 +948,10 @@ export function isAffirmative(parameter: any): boolean {
   return ['1', 'on', 'true', 'yes'].includes(String(parameter).toLowerCase());
 }
 
+export function isNotAffirmative(parameter: any): boolean {
+  return ['0', 'off', 'false', 'no'].includes(String(parameter).toLowerCase());
```

Comment:
Aghhh, I feel very stressed about this function, and have a strong opinion to remove it.
```
isAffirmative("") == isNotAffirmative("")
```
It makes both those function useless in my opinion, as I can't reason about it without running the code somewhere else.